CrossPlatform.sln

```diff
@@ -144,6 +144,14 @@ Global
 		{1EE8CAD3-55F9-4D91-96B2-084641DA9A6C}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{1EE8CAD3-55F9-4D91-96B2-084641DA9A6C}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{1EE8CAD3-55F9-4D91-96B2-084641DA9A6C}.Release|Any CPU.Build.0 = Release|Any CPU
+		{9508F118-F62E-4C16-A6F4-7C3B56E166AD}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Err, forgot to hit add comment on this - this is VBCSCompiler and VBCSCompilerTests, should have been added in https://github.com/dotnet/roslyn/pull/22142